ui/apps/ui/src/app/components/filters/filter-multiselect/utils.ts

```diff
@@ -43,3 +43,13 @@ export const search = (query: string | null, entities: IFilterNode[]) => {
     .search(query)
     .map(({ item }) => item);
 };
+
+export const query_changer = (q: string) => {
+  const nq = q.split(' ');
+  let new_query = '';
+  nq.forEach(function (value) {
+    value = value + '~1';
+    new_query = new_query + value + ' ';
+  });
+  return new_query;
+};
```

Comment:
1. We are using `camelCase` in the frontend, so `query_changer` should be `queryChanger`
2. Maybe instead of using `nq`, you should use `mutatedQuery`, `changedQuery`
3. You're code can be simplified significantly like

```javascript
export const queryChanger = (q: string): string => q.split(' ')
  .map((word) => `${word}~1`)
  .join(' ');
```

4. As you're using the method in multiple places it should be exported from some `utils.ts` file. You can put the code into `ui/apps/ui/src/app/collections/filters-serializers/utils.ts`.